hms-app/src/components/RelevantIdeasLoader.tsx

```diff
@@ -77,14 +77,17 @@ export default function RelevantIdeasLoader({
   }
 
   useEffect(() => {
-    loadSelectedHackathonData()
-  }, [selectedHackweek])
+    localStorage.getItem(selectedHackathonID)
```

Comment:
You won't need this if you use the custom hook, as it will be handled by default